spark/hbase-spark/src/test/scala/org/apache/hadoop/hbase/spark/DefaultSourceSuite.scala

```diff
@@ -1030,6 +1033,7 @@ class DefaultSourceSuite
     assert(Math.abs(result.get(8).asInstanceOf[Date].getTime - timestamp) <= 86400000)
     assert(result.get(9).asInstanceOf[Timestamp].getTime == timestamp)
     assert(result.get(10) == "string")
+    assert(result.get(11) == decimal)
```

Comment:
can we also assert on value here?

Reply:
This is already assert on value....

Reply:
Ah my bad, I confused we are assessing data type here. Thanks for the clarification.